src/app/pages/create-wallet/create-wallet.scss

```diff
@@ -0,0 +1,92 @@
+@import 'preact-material-components/Card/mixins';
+@import 'preact-material-components/Button/mixins';
+@import '../../styles/base';
+
+$mdc-layout-grid-breakpoints: (
+    desktop: 512px,
+    tablet: 280px,
+    phone: 0px
+);
+
+@import '~preact-material-components/LayoutGrid/mdc-layout-grid.scss';
```

Comment:
move this import in `src/app/styles/material-componens.scss`, it's the global import for material components